.buildkite/engineer

```diff
@@ -13,7 +13,7 @@ fi
 if ! type "engineer" > /dev/null; then
     # Setup Prisma engine build & test tool (engineer).
     set -e
-    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/1.48.0/latest/$OS/engineer.gz" --output engineer.gz
+    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/cdb-versioning/latest/$OS/engineer.gz" --output engineer.gz
```

Comment:
Note to fix this before merge

Reply:
[`ba1ebcc` (#3742)](https://github.com/prisma/prisma-engines/pull/3742/commits/ba1ebcc90a1749d5cf044b04c5a3329ccb5a31a2)